main.py

```diff
@@ -148,5 +149,76 @@ async def keen(context):
     await context.send(result)
 
 
-bot.run(TOKEN)
+@bot.command(name='daily', help='Will give the daily leaderboard for specified day')
+async def daily(context, day : str = str(datetime.datetime.today().day)):
```

Comment:
Replacing:
`str(datetime.datetime.today().day)`
with
`str(datetime.datetime.today() - datetime.timedelta(hours=5)).day`
should hopefully compensate for the time difference. The server time is in UTC, so this should convert it to the UTC-5 the challenges come out in. A comment to this effect would be useful too (on the line below this, i.e. the first line of the function)

Reply:
Sorted if you can just check the comment for me